### PR TITLE
update animation tutorial images

### PR DESCRIPTION
**Pull request by skyace65**
Updates the animation tutorial images, the old images were for 3.0 and the new images are for 3.1. Also updated the tutorial in one section for 3.1. All images have gone through png compression. Closes Issue #2266

### CODE REVIEW

**Comment by merumelu** [comment]
Can you redo the images of the animation panel so they are less wide, please? The content div for the docs is styled with `max-width: 800px` so these images would all get resized by the browser to fit which might make them somewhat blurry and text harder to read.

**Comment by skyace65** [comment]
Done. All images are the proper width now, and once again everything has been run through PNG compression.

File: getting_started/step_by_step/animations.rst [hunk 1]
```diff
@@ -30,8 +30,11 @@ The animation editor panel stays visible until manually hidden.
 Creating the animation
 ----------------------
 
-It's time to create a new animation! Press the new animation button and name
-the animation "intro" when the dialog appears.
+It's time to create a new animation! Press the animation button then select
+the new animation button from the menu. name the animation "intro" when
```
Comment (cbscribe) [inline getting_started/step_by_step/animations.rst]: Too many "buttons". You click the "Animation" button (the name is capitalized" and select "New" from the menu, not another button.

"Name" should be capitalized at the beginning of the third sentence.

**Comment by skyace65** [comment]
Should be fixed now.

**Comment by cbscribe** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.

**Comment by golddotasksquestions** [comment]
This issue still persists:
https://docs.godotengine.org/en/3.1/tutorials/animation/introduction_2d.html
https://www.reddit.com/r/godot/comments/bzgxi9/help_why_does_my_animation_panel_not_look_like/

**Pull request merged**